.github/workflows/publish_ground_control_client.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish client
+
+on:
+  push:
+    tags:
+    - 'ground_control_client-v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  publish:
+    permissions:
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dart-lang/setup-dart@v1
+      - name: Install dependencies
+        run: dart pub get
+        working-directory: ground_control_client
+      - name: Publish
+        run: dart pub publish -force --skip-validation
```

Comment:
`--force` is required to bypass interactive mode

`--skip-validation` is required to not check that all dependency versions exist when uploading, this is a problem when uploading both of these at the same time, as there is a dependency from the cli to the client. According to pub.dev this is the way to get around that issue.